### PR TITLE
93 Adding new JAX center frequencies to vERAM, vSTARS and VRC POF

### PR DESCRIPTION
**Pull request by patrgl**
Added the positions for the POF, vERAM, vSTARs (PBI, TPA) files

### CODE REVIEW

File: docs/pof/ZMA.pof [hunk 1]
```diff
@@ -233,23 +233,29 @@ NQX_GND:Navy Key West Ground:121.700:GX:G:NQX:GND:NQX:GND:1001:1077
 
 
 ; JACKSONVILLE CENTER HIGH SECTORS (J##)
-JAX_30_CTR:Jacksonville Center:124.770:J30:J:JAX:CTR:JAX:CTR:1001:1047
-    JAX_30_CTR:Jacksonville Center:124.775:J30:J:JAX:CTR:JAX:CTR:1001:1047
+JAX_11_CTR:Jacksonville Center:124.475:J11:J:JAX:CTR:JAX:CTR:1001:1047
+JAX_16_CTR:Jacksonville Center:125.175:J16:J:JAX:CTR:JAX:CTR:1001:1047
```
Comment (erikquinn) [inline docs/pof/ZMA.pof:237]: Compared to the above, can you add secondary entries for any position whose frequency ends in a .x2 or .x7 to include both the .x20 and .x25 (etc) variations?

Really, people should know to use .175, but it doesn't always happen. And if it's defined in the POF as .175 and they prim on .170, the client doesn't think it is this sector anymore. By including the same line twice, once with .175 and once with .170, the handoff code for J16 will always be `J16`, regardless of whether their controller prims on .170 or .175.

**Review by erikquinn — CHANGES_REQUESTED** [review]
Also looks like the vERAM/vSTARS files don't match what's in the POF currently (I see two entries for J30, for example).

File: docs/pof/ZMA.pof (hunk 1 above)
Comment (patrgl) [reply docs/pof/ZMA.pof:237]: Ahh completely glossed over that! I'll put that in later tonight :)

File: docs/pof/ZMA.pof [hunk 2]
```diff
@@ -233,23 +233,37 @@ NQX_GND:Navy Key West Ground:121.700:GX:G:NQX:GND:NQX:GND:1001:1077
...
+	JAX_16_CTR:Jacksonville Center:125.170:J16:J:JAX:CTR:JAX:CTR:1001:1047
+JAX_30_CTR:Jacksonville Center:135.925:J30:J:JAX:CTR:JAX:CTR:1001:1047
+	JAX_30_CTR:Jacksonville Center:135.920:J30:J:JAX:CTR:JAX:CTR:1001:1047
 JAX_33_CTR:Jacksonville Center:125.050:J33:J:JAX:CTR:JAX:CTR:1001:1047
 JAX_35_CTR:Jacksonville Center:134.850:J35:J:JAX:CTR:JAX:CTR:1001:1047
-JAX_51_CTR:Jacksonville Center:120.120:J51:J:JAX:CTR:JAX:CTR:1001:1047
-    JAX_51_CTR:Jacksonville Center:120.125:J51:J:JAX:CTR:JAX:CTR:1001:1047
-JAX_66_CTR:Jacksonville Center:127.870:J66:J:JAX:CTR:JAX:CTR:1001:1047
-    JAX_66_CTR:Jacksonville Center:127.875:J66:J:JAX:CTR:JAX:CTR:1001:1047
+JAX_50_CTR:Jacksonville Center:135.970:J50:J:JAX:CTR:JAX:CTR:1001:1047
```
Comment (erikquinn) [inline docs/pof/ZMA.pof:244]: 135.975 / 135.970

**Review by erikquinn — CHANGES_REQUESTED** [review]
Missed a spot! Otherwise looks good to me

File: docs/pof/ZMA.pof (hunk 2 above)
Comment (erikquinn) [reply docs/pof/ZMA.pof]: done

**Review by erikquinn — APPROVED** [review]
Looks good to me now.

@patrgl Just an FYI, be very careful editing the vSTARs/vERAM files by hand. It's easy enough to do, but leads to the possibility of breaking. Where you had `adjacent` instead of `Adjacent`, it broke the file and vSTARS couldn't load it because it was corrupt. So I'd recommend the (more tedious, but more reliable) approach of 7zip to *.gz, import, edit in the menus, export, 7zip to "extract here".

**Pull request merged**